compiler/rustc_ast/src/ast.rs

```diff
@@ -3097,6 +3097,7 @@ impl CoroutineKind {
 #[derive(Copy, Clone, PartialEq, Eq, Hash, Encodable, Decodable, Debug)]
 #[derive(HashStable_Generic, Walkable)]
 pub enum Const {
+    Comptime(Span),
     Yes(Span),
```

Comment:
Would be nice to some day make all the variant names meaningful. Something like 

```rust
enum EvaluationTime {
    Comptime,
    MaybeComptime,
    Runtime,
}
```

Looking at your PR yday that's one of the first things I thought. First decided not to comment it because it might not be for this PR but coming back I thought I'd mention it anyway.